src/index.html

```diff
@@ -1,12 +1,444 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="page-header">
+      <a class="page-header__link-logo" href="#home">
+        <img
+        src="./images/logo.png"
+        alt="LogoMoyo"
+        class="page-header__link-images"
+        >
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="LogoMoyo"
          class="page-header__link-images"
        >
```

Reply:
not fixed
https://github.com/mate-academy/layout_catalog/pull/3299/files#r1226668584